lib/redux/middleware/sentry_api_middleware.dart

```diff
@@ -30,9 +32,20 @@ class SentryApiMiddleware extends MiddlewareClass<AppState> {
           var currentProgress = 0;
 
           for (final organization in organizations) {
-            final individualOrganization =
-                await api.organization(organization.slug);
-            individualOrganizations.add(individualOrganization);
+            final organizationSpan =
+                transaction.startChild('task', description: 'organization');
```

Comment:
`task` is a very generic name, I'd rather do `organization` for operation name then and a more descriptive description